src/instrumentation.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  exportApiIdentifier,
+  exportFunctionIdentifier,
+  verificationApiIdentifier,
+  verificationTableIdentifier,
+  verifiedTableIdentifier,
+} from 'src/app/regionen/[regionSlug]/_mapData/mapDataSources/sources.const'
+import { prismaClientForRawQueries } from 'src/prisma-client'
+
```

Comment:
Ich denke, die Datei sollte noch eine Einleitung bekommen. Sowas würde ich sonst in ein README im jew. Ordner packen, aber da die Datei im root liegt…

```suggestion

/**
 * File is executed once when the server is started.
 * Docs: https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation
 * This file will setup our DB
 * 1. `await initExportFunctions(exportApiIdentifier)`
      Create the PostgreSQL functions that are used by the export API in `src/pages/api/export/[tableName].ts`
 * 1. `await initVerificationViews(verificationApiIdentifier)`
      Create the post commit hook, tables and indexes that are required for our verification feature.
*/

```